docs/source/advanced/backends.rst

```diff
@@ -1,14 +1,108 @@
+    | Execution strategy grid-blocks                                | sequential                                    | sequential                                                                    | sequential                                                                   | sequential                                                                      | undefined                                                                                                                          | undefined                                        |
+    +---------------------------------------------------------------+-----------------------------------------------+-------------------------------------------------------------------------------+------------------------------------------------------------------------------+---------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------+
+    | Execution strategy block-kernels                              | sequential                                    | preemptive multitasking                                                       | cooperative multithreading                                                   | preemptive multitasking                                                         | preemptive multitasking                                                                                                            | lock-step within warps                           |
+    +---------------------------------------------------------------+-----------------------------------------------+-------------------------------------------------------------------------------+------------------------------------------------------------------------------+---------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------+
+    | getIdx                                                        | n/a                                           | block-kernel: mapping of std::thisthread::getid() grid-block: member variable | block-kernel: mapping of std::thisfiber::getid() grid-block: member variable | block-kernel: ompgetthreadnum() to 3D index mapping grid-block: member variable | block-kernel: ompgetthreadnum() to 3D index mapping grid-block: member variable                                                    | threadIdx, blockIdx                              |
```

Comment:
Why is `getIdx` for `Serial` marked as `n/a`? I think that gives a wrong impression that it's not useable.

Reply:
I would replace it with: `no mapping: every time id 0`